### PR TITLE
DO NOT MERGE - Benchmarks with dummy signature scheme

### PR DESCRIPTION
**Pull request by jonas-lj**
This is not supposed to be merged. The PR is created to test the code in CI before it's deployed to testnet.

### CODE REVIEW

**Review by kchalkias — COMMENTED** [review]
Ooc, is there a better pattern (ie load the key type from some config) to be able to switch key types without changing the code each time?

yeah it seems a bit more complex as we pin validator sig Vs being abstracted, maybe it's a feature as we should be careful with that and when we agree on the scheme we won't change it. All good, thanks for this PR, very useful for benchmarking

**Comment by jonas-lj** [comment]
> Ooc, is there a better pattern (ie load the key type from some config) to be able to switch key types without changing the code each time?
> 
> yeah it seems a bit more complex as we pin validator sig Vs being abstracted, maybe it's a feature as we should be careful with that and when we agree on the scheme we won't change it. All good, thanks for this PR, very useful for benchmarking

We could add the extra functions under a feature flag or something like that, but it would add quite a bit of code that is redundant when not running with unsecure signatures, so I think having this PR and keeping it up to date every few days is probably just as easy.

File: crates/mysten-util-mem/src/external_impls.rs [hunk 1]
```diff
@@ -13,6 +13,9 @@ malloc_size_of_is_0!(fastcrypto::bls12381::min_sig::BLS12381AggregateSignature);
 malloc_size_of_is_0!(fastcrypto::bls12381::min_pk::BLS12381PublicKey);
 malloc_size_of_is_0!(fastcrypto::bls12381::min_pk::BLS12381Signature);
 malloc_size_of_is_0!(fastcrypto::bls12381::min_pk::BLS12381AggregateSignature);
+malloc_size_of_is_0!(fastcrypto::unsecure::signature::UnsecurePublicKey);
```
Comment (kchalkias) [inline crates/mysten-util-mem/src/external_impls.rs:16]: Let's rename this actually to Insecure from Unsecure (irrelevant to that draft PR)

**Review by kchalkias — COMMENTED** [review]
This PR should be shared among any benchmark dev, so we know how to properly use it. Publishing a super short tutorial re how one could use Insecure sigs for benchmarking and making it clear that this OR should never be merged would help.

**Pull request closed without merging**